pkg/kudoctl/packages/resolver/resolver_url.go

```diff
@@ -4,29 +4,31 @@ import (
 	"bytes"
 	"fmt"
 
+	"github.com/spf13/afero"
+
 	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
 	"github.com/kudobuilder/kudo/pkg/kudoctl/http"
 	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
 	"github.com/kudobuilder/kudo/pkg/kudoctl/packages/convert"
 	"github.com/kudobuilder/kudo/pkg/kudoctl/packages/reader"
 )
 
-// URLResolver will resolve a packages from a url
-type URLResolver struct {
+// URLHelper will resolve a packages from a url
+type URLHelper struct {
 	client http.Client
 }
 
 // Resolve returns a package for the provided url
-func (f *URLResolver) Resolve(name string, appVersion string, operatorVersion string) (*packages.Resources, error) {
-	// check to see if name is url
-	if !http.IsValidURL(name) {
-		return nil, fmt.Errorf("resolver: url %v invalid", name)
+func (f *URLHelper) Resolve(out afero.Fs, url string) (*packages.Resources, error) {
```

Comment:
`Resolve` is overloaded. Let's rename this to `ResolveURL`, similar to the methods in `LocalHelper`. Also, `out` should be a return value.